package.json

```
@@ -11,6 +14,7 @@
     "eslint-plugin-standard": "^4.0.1"
   },
   "scripts": {
-    "lint": "eslint lib/js/*.js --fix"
+    "lint": "eslint lib/js/macaroons.js --fix && eslint lib/js/sku.js --fix",
+    "build": "browserify lib/js/sku.js --standalone braveSku > lib/js/sku-lib.js"
```

Let's name this `BraveSKU`, usually libraries start with uppercase and SKU is an acronym. 👍 
```suggestion
    "build": "browserify lib/js/sku.js --standalone BraveSKU > lib/js/sku-lib.js"
```